src/Form.js

```diff
@@ -100,6 +100,25 @@ export default class Form {
   }
 
   /**
+   * Checks if all fields have a value.
+   * @type {boolean}
+   */
+  @computed get isComplete (): boolean {
+    const value = _.find(this.fields, (field: Field) => {
```

Comment:
from lodash docs: 

`Iterates over elements of collection, returning the first element predicate returns truthy for.` 

Are you sure it iterates over all fields? If it's like the [oficial implementation of the method find](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find), it should not.